src/Printer/Console.php

```diff
@@ -4,52 +4,83 @@
 
 namespace Povils\PHPMND\Printer;
 
-use JakubOnderka\PhpConsoleColor\ConsoleColor;
 use JakubOnderka\PhpConsoleHighlighter\Highlighter;
-use Povils\PHPMND\FileReportList;
+use Povils\PHPMND\DetectionResult;
 use Povils\PHPMND\HintList;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class Console implements Printer
 {
     private const DEFAULT_LINE_LENGTH = 80;
 
-    public function printData(OutputInterface $output, FileReportList $fileReportList, HintList $hintList): void
+    /**
+     * @var Highlighter
+     */
+    private $highlighter;
+
+    public function __construct(Highlighter $highlighter)
+    {
+        $this->highlighter = $highlighter;
+    }
+
+    public function printData(OutputInterface $output, HintList $hintList, array $list): void
```

Comment:
what is list here ? Can we name it better?

Reply:
done